src/schema/objects/enums.yaml

```
@@ -1436,3 +1436,90 @@ microvascular:
   display_name: microvascular
   description: |
     The origin of a tissue: microvascular
+start_of_embryo_stage:
+  value: start of embryo stage
+  display_name: start of embryo stage
+  description: |
+    Point of fertilization marking the beginning of the subject's embryo
+    stage.
```

```suggestion
start_of_fertilization:
  value: start of fertilization stage
  display_name: start of fertilization stage
  description: |
    Point of fertilization marking the beginning of the subject's embryonic
    stage.
```

@lzehl How about this?